tasks/openjdk.yml

```diff
@@ -1,4 +1,8 @@
 ---
+- name: Add OpenJDK PPA
+  apt_repository: repo='ppa:openjdk-r/ppa'
+  when: "{{ ansible_distribution_version | version_compare('14.04', '=') }}"
```

Comment:
I think that we also want to suppress this task in the event that the user wants to install OpenJDK 7.